lib/pengine/pe_output.c

```diff
@@ -11,6 +11,58 @@
 #include <crm/common/iso8601_internal.h>
 #include <crm/pengine/internal.h>
 
+char *
+pe__node_display_name(node_t *node, bool print_clone_detail)
```

Comment:
In the context of this function, it doesn't have anything to do with clones, it's just a detail flag